pkgs/applications/editors/neovim/utils.nix

```diff
@@ -13,46 +13,11 @@
 let
   inherit (vimUtils) toVimPlugin;
 
-   /* returns everything needed for the caller to wrap its own neovim:
-   - the generated content of the future init.vim
-   - the arguments to wrap neovim with
-   The caller is responsible for writing the init.vim and adding it to the wrapped
-   arguments (["-u" writeText "init.vim" GENERATEDRC)]).
-   This makes it possible to write the config anywhere: on a per-project basis
-   .nvimrc or in $XDG_CONFIG_HOME/nvim/init.vim to avoid sideeffects.
-   Indeed, note that wrapping with `-u init.vim` has sideeffects like .nvimrc wont be loaded
-   anymore, $MYVIMRC wont be set etc
-   */
-  makeNeovimConfig =
```

Comment:
Is this removed? Could you add a deprecation stub that throws an appropriate error message for any publicly exposed functions/etc that have been removed?

Reply:
Ah, my mistake. This is not removed. Is anything else removed without a `throw`?